src/Psalm/Storage/ClassLikeStorage.php

```diff
@@ -183,13 +183,18 @@ final class ClassLikeStorage implements HasAttributesInterface
      */
     public $trait_alias_map = [];
 
+    /**
+     * @var array<string, string>
+     */
+    public array $trait_alias_map_cased = [];
+
     /**
      * @var array<lowercase-string, bool>
      */
     public $trait_final_map = [];
 
     /**
-     * @var array<string, int>
+     * @var array<string, 1|2|3>
```

Comment:
ClassLikeAnalyzer::VISIBILITY_* would fit better here

Reply:
Psalm says that is an invalid doc-block, so I resorted to this instead